Update Introduction.md (#833)
wix-incubator/wix-rest-docs: all/guides/wix-groups/docs/Flow.md

> For example, create a group for each class at a school or each fitness class (yoga, TRX, etc) in a gym.

Reviewer: Where is the standard intro paragraph for flow articles?

> 1. Create a list of site members to add to the relevant group, identified by one of the following: name, email address, phone number, contactId.

Reviewer: ```suggestion
1. Create a list of site members to add to the relevant group, identified by one of the following: name, email address, phone number, contact ID.
```

2. Create a group by calling [Create Group](https://dev.wix.com/docs/rest/crm/community/groups/groups/create-group).
3. Call [Query Members](https://dev.wix.com/docs/rest/crm/members-contacts/members/members/query-members) to get the site member data.
4. Map the site members to add to the group to their IDs as returned in step 3.
5. Add the site members to the group by calling [Add Group Members](https://dev.wix.com/docs/rest/crm/community/groups/members/add-group-members) with the site member IDs collected above.

> ## Manage Bulk Emailing to Group Members

Author: ```suggestion
## Manage bulk emailing to group members
```

## Send Promotional Emails Based on Members' Interests

Your can identify group members who are interested in a certain topic by searching for keywords in their answers to the membership questions.
Then, you can send emails to the identified members informing them about upcoming events or special offers.

1. Add membership questions by calling [Create or Replace All Membership Questions](https://dev.wix.com/docs/rest/crm/community/groups/membership-questions/create-or-replace-all-membership-questions).
2. At any time you want to send emails to selected members, call [List Answers](https://dev.wix.com/docs/rest/crm/community/groups/membership-questions/list-answers).
3. Identify the relevant members via a keyword search.
4. Call [Query Email Subscriptions](https://dev.wix.com/docs/rest/crm/communication/email-subscriptions/query-email-subscriptions) to confirm that the identified members have agreed to receive emails from the site owners.
5. Prepare and send the automated emails.

## Bulk Update Group Rules

When Wix users manually update the rules for one group, You can apply the new rules to other relevant groups.

> 1. Get the updated group rules by calling [List Rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/list-rules).

Reviewer: Wouldn't you want to start by listening for a change event or something like that?

Author: ```suggestion
1. Listen to [Rule Created](https://dev.wix.com/docs/rest/crm/community/moderation-rules/rule-created) and [Rule Updated](https://dev.wix.com/docs/rest/crm/community/moderation-rules/rule-updated), and or regularly get the updated group rules by calling [List Rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/list-rules).
```
